perses/samplers/samplers.py

```diff
@@ -861,28 +239,45 @@ def get_log_weight(self, state_key):
             self.log_weights[state_key] = 0.0
         return self.log_weights[state_key]
 
-    def _geometry_forward(self, topology_proposal, old_positions):
+    def _system_to_thermodynamic_state(self, system):
```

Comment:
This method is only used once and seems like a silly thing to factor out given the creation of a `ThermodynamicState` is a one-liner.

Reply:
changed.